src/main/java/at/hannibal2/skyhanni/utils/tracker/SkyHanniTracker.kt

```diff
@@ -99,14 +99,14 @@ open class SkyHanniTracker<Data : TrackerData>(
         dirty = true
     }
 
-    private fun buildFinalDisplay(searchBox: Renderable) = buildList {
+    private fun buildFinalDisplay(searchBox: Renderable, displayModeToggleable: Boolean) = buildList {
         add(searchBox)
         if (isEmpty()) return@buildList
-        if (inventoryOpen) {
+        if (inventoryOpen && displayModeToggleable) {
             add(buildDisplayModeView())
-        }
-        if (inventoryOpen && getDisplayMode() == DisplayMode.SESSION) {
-            add(buildSessionResetButton())
+            if (getDisplayMode() == DisplayMode.SESSION) {
+                add(buildSessionResetButton())
+            }
```

Comment:
since CakeTracker isnt using skyhannitracker anymore, you should probably remove these changes from this pr